### PR TITLE
Feat/notifications: added pages for notifications

### PR DESCRIPTION
**Pull request by amandadupell**
#### What does this PR do?

Adds the main [Notifications](https://deploy-preview-1770--keen-mayer-a86c8b.netlify.app/components/notifications) page, the [Toast Notifications](https://deploy-preview-1770--keen-mayer-a86c8b.netlify.app/components/toast-notifications) subpage, and the [Status Indicator](https://deploy-preview-1770--keen-mayer-a86c8b.netlify.app/components/status-indicator) page

#### Where should the reviewer start?

Go into the components sections and explore all 3 new pages.

#### What testing has been done on this PR?

Visual testing locally.

#### How should this be manually tested?

Run the app locally.

#### Should this PR be mentioned in Design System updates?

Yes

### CODE REVIEW

**Comment by halocline** [comment]
@amandadupell - FYI, my 👀  are on this and I have some review comments in progress, but may not have them wrapped up by end of day.

File: aries-site/src/pages/components/status-indicator.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## What makes up a status indicator?
+
+To effectively capture the user’s attention, status indicators are compromised of four different elements:
+
+- Colors
+- Icons
+- Shapes
+- Content
+
+To meet WCAG 2.1 compliance, at least three out of four elements must always be present.
```
Comment (taysea) [inline aries-site/src/pages/components/status-indicator.mdx:17]: Can we link to the compliance guidance here?

File: aries-site/src/pages/components/status-indicator.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- A critical status used to alert a device failure
+- A warning status used to notify the user of an expiring license
+- A normal status used to communicate that an update has successfully completed
+- An unknown status used to indicate that a page cannot be found
+
+### Icons and Shapes
+
+Icons and shapes are used to represent the notification’s level of severity to users. Here are two sets of icons and shapes created for Statuses found within the [HPE Design System Icons](https://design-system.hpe.design/foundation/icons) page.
+
+<Box direction="row" gap="small" pad={{bottom: 'medium'}}>
```
Comment (taysea) [inline aries-site/src/pages/components/status-indicator.mdx:34]: It might be nice to label each shape with the associated status

File: aries-site/src/pages/components/status-indicator.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    <StatusWarningSmall color="status-warning" />
+    <StatusCriticalSmall color="status-critical" />
+    <StatusUnknownSmall color="status-unknown" />
+</Box>
+
+*** Remember that WCAG 2.1 guidlines states that at least three out of four elements must be present, therefore, this sets will not pass accessibility requirements if it is not paired with another element such as well-written and relevant content. 
+
+### Content
+
+Do This:
```
Comment (taysea) [inline aries-site/src/pages/components/status-indicator.mdx:45]: can we bold the "Do This" / "Don't Do This" sections?

File: aries-site/src/pages/components/status-indicator.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- When constructing Content be sure to use words, phrases, and concepts familiar to the user, rather than internal jargon.
```
Comment (taysea) [inline aries-site/src/pages/components/status-indicator.mdx:46]:
```suggestion
- When constructing content be sure to use words, phrases, and concepts familiar to the user, rather than internal jargon. 
```
Since "Content" isn't an actual component, would it make more sense for it just to be lowercase?

File: aries-site/src/pages/components/notifications.mdx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+### What are they for?
+
+Notifications build trust by providing users confidence and assurance that they are informed with timely, relevant status about their systems and tasks.
+
+## Taxonomy 
+
+To standardize our language on Notifications across platforms, be sure to familiarize yourself with this taxonomy:
+
+- Type - In the broadest terms, what the notification is about
```
Comment (taysea) [inline aries-site/src/pages/components/notifications.mdx:12]: Might be helpful to bold these for a little extra contrast
```suggestion
- **Type** - In the broadest terms, what the notification is about
```

File: aries-site/src/pages/components/notifications.mdx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+- State - The current stage within a process
+- Status - The condition of a process
+- Attention Level - The priority and importance of a notification
+- Actions - Interactactions within a notification
+- Persistence - How the notification is dismissed
+- Placement - Where the notification can be found
+- Insight - The content of the notification
+
+<Box direction="row-responsive" gap="medium" pad={{ top: 'medium' }}>
```
Comment (taysea) [inline aries-site/src/pages/components/notifications.mdx:21]: Maybe we could implement this as a markdown table so it looks more like the tables on the Voice and Tone page. Might also cleanup the code for this page a little. Documentation: https://www.markdownguide.org/extended-syntax/#tables

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+## Guidance
+
+Everything that you need to know about toasts and more.
+
+### When should you use a Toast Notification?
+
+Toasts should be considered when displaying these types of information to the user:
+
+- Low attention messages that do not require user action
+- Singlular status updates
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:28]:
```suggestion
- Singular status updates
```

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+### Content (Title + Message)
+
+All toasts have content which is comprised of a title and a message. However, the content only requires a title, not necessarily a message. The title and message should be distinguishable from each other by the weight of the text. The status title and message should be distinguishable by the weight of the text.
+
+1. Amount of Information
+
+Toast notifications should not have too much information inside of them. Additionally, each notification should only revolve around one idea or subject. Because a toast notification is only present on the screen for a short amount of time, a user using assistive technology needs to have an understanding of what information is in the notification. Therefore, the content should be minimal and direct.
+
+For best practice, keep the amount of information to three lines or under. This three line limit either can be a combination of both the title and the message or a three-lined title. If the information is heavy and long, consider using a different type of notification other than toast. NEVER truncate text within a toast. The main point is to keep the information concise and clear.
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:77]: Typically, we try to avoid using all caps and just stick with sentence case.

```suggestion
For best practice, keep the amount of information to three lines or under. This three line limit either can be a combination of both the title and the message or a three-lined title. If the information is heavy and long, consider using a different type of notification other than toast. Never truncate text within a toast. The main point is to keep the information concise and clear.
```

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+### Close Button
+
+The close button within a toast should be the only interactable element and is used to dismiss the notification. To clarify, the toast itself is not dismissable on click rather only when the close button is specifically activated. This is to avoid any confusion the user may have on what the interactable elements are within a toast.
+
+## Placement 
+
+Make sure to follow these guidelines for Toast Notifications:
+- Place them either top-center or top-right of the screen. If badging is being used, toasts should be placed underneath it. Note that top-right placements may negatively affect those using screen magnifiers.
+- Fade in on arrival and fade out when dismissed.
+- Do not obscure critical information and any primary action elements.
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:92]: Agreed with this point. However, often times, page level actions live at the top right of our layouts. Will this conflict? Or, where should toasts go and how do we know what kind of layout a user is going to be on when a toast appears?

File: aries-site/src/data/structures/components.js [hunk 10]
```diff
@@ -410,6 +434,18 @@ export const components = [
     relatedContent: ['Lists', 'Table', 'Forms'],
     sections: [],
   },
+  {
+    name: 'Status Indicator',
+    category: 'Visualizations',
+    description: 'Utilized to highlight notification messages and alerts the users to anything that might require their attention.',
```
Comment (halocline) [inline aries-site/src/data/structures/components.js:440]:
```suggestion
    description: 'Quickly provide peace-of-mind or call attention to items requiring a user's action.',
```

File: aries-site/src/data/structures/components.js [hunk 11]
```diff
@@ -410,6 +434,18 @@ export const components = [
...
+    preview: {
+      component: () => <StatusIndicatorPreview />,
+      background: 'background-front',
+    },
+    seoDescription: 'Utilized to highlight notification messages and alerts the users to anything that might require their attention.',
```
Comment (halocline) [inline aries-site/src/data/structures/components.js:445]:
```suggestion
    seoDescription: 'Highlight notification messages and alerts which require a user's attention. Status indicators provide peace-of-mind when all is well or call attention to items when a user needs to take action',
```

File: aries-site/src/data/structures/components.js (hunk 10 above)
Comment (halocline) [reply aries-site/src/data/structures/components.js:440]: Are status indicators solely for use with notifications and alerts? Can they be used in tables, lists, etc.? If more broadly, perhaps we can focus on the quickly communicate detail?

File: aries-site/src/data/structures/components.js [hunk 12]
```diff
@@ -352,7 +354,29 @@ export const components = [
     },
     relatedContent: ['TextInput', 'Forms', 'Select'],
   },
-
+  {
+    name: 'Notifications',
```
Comment (halocline) [inline aries-site/src/data/structures/components.js:358]: Wondering if "Notification" should be under components and "Notifications" and "Status Indicators" under templates? Notifications being why and where these patterns applied within an application and then would reference "Notification" component page for the various flavors.

This is a bit of wondering aloud for discussion purposes. No changes for this in this PR.

**Review by halocline — COMMENTED** [review]
dropping comment incrementally

File: aries-site/src/examples/cardPreviews/toast-notification.js [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+                    color={{ light: 'black', dark: 'white' }}
+                >Hooray!</Text>
+                <Text 
+                    color={{ light: 'black', dark: 'white' }}
+                >Your toast is done!</Text>
+            </Box>
+            <Button
+              icon={<FormClose color={{ light: 'black', dark: 'white' }} />}
+              plain
+            />
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/toast-notification.js:38]: Eliminate double focus / button within a button:
Screen Shot 2021-08-13 at 2 59 21 PM


```suggestion
            <Button
              icon={<FormClose color={{ light: 'black', dark: 'white' }} />}
              plain
              tabIndex={-1}
            />
```

File: aries-site/src/examples/components/toast-notifications/ToastNotification.js [hunk 14]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { FormClose } from 'grommet-icons/icons/FormClose';
+
+import { Box, Button, Layer, Text } from 'grommet';
+
+const Notification = ({ toast, title, message, onClose }) => {
```
Comment (halocline) [inline aries-site/src/examples/components/toast-notifications/ToastNotification.js:8]: Keep name of export and file name consistent? Style/convention preference.

```suggestion
const ToastNotification = ({ toast, title, message, onClose }) => {
```

Comment (halocline) [reply aries-site/src/examples/components/toast-notifications/ToastNotification.js:8]: Or alternatively this file could be called `Notification.js`. Presumably this file becomes obsolete once Notification is available by Grommet.

File: aries-site/src/examples/components/toast-notifications/ToastNotification.js [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+              <Text 
+                  weight="bold" 
+                  color={{ light: 'black', dark: 'white' }}
+              >{title}</Text>
+            {message && (
+              <Text 
+                  color={{ light: 'black', dark: 'white' }}
+              >{message}</Text>
+            )}
+          </Box>
```
Comment (halocline) [inline aries-site/src/examples/components/toast-notifications/ToastNotification.js:39]:
```suggestion
          <>
              <Text 
                  weight="bold" 
                  color={{ light: 'black', dark: 'white' }}
              >{title}</Text>
            {message && (
              <Text 
                  color={{ light: 'black', dark: 'white' }}
              >{message}</Text>
            )}
          </>
```

File: aries-site/src/examples/cardPreviews/toast-notification.js [hunk 16]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { FormClose } from 'grommet-icons/icons/FormClose';
+
+import { Box, Button, Text } from 'grommet';
+
+export const ToastPreview = () => (
+    <Box align="center" justify="center" fill background="background-back">
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/toast-notification.js:8]: Will this get replaced with a `<Notification toast />` component once available from Grommet? If so, perhaps adding a TODO comment "replace this with ... when Notification component is available in Grommet" and/or add an issue to track this?

Comment (halocline) [reply aries-site/src/examples/cardPreviews/toast-notification.js:8]: Could you import Notification from `toast-notifications/ToastNotification.js` in order to re-use code? Looks very similar.

Basically thinking that if we are mimicking the implementation of the Notification component until it is available, can we mimic it in a single place, have all current imports point to it? Then when Notification is available, we simply replace the import from the local file to the import from Grommet?

File: aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+          <Box 
+            id="3" 
+            border 
+            round="full" 
+            pad={{ horizontal: 'small', vertical: 'xsmall' }}
+          >3</Box>
+        </Box>
+        <Notification title="Status Title" message="This is a status message" onClose />
+      </Box>
+      <Diagram connections={connections} />
```
Comment (halocline) [inline aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js:104]: Nice! 👏

File: aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+        anchor,
+        type,
+        color,
+        thickness,
+        fromTarget: '3',
+        toTarget: 'close-button',
+    },
+];
+
+export const ToastDiagram = () => (
```
Comment (halocline) [inline aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js:79]: Style/convention comment. Move to its own contained file and leave ToastNotificationExample only showing the example?

Also, the intent of the examples are for developers to be able to copy/paste the code example into their project. The toast diagram piece here would be extraneous, correct?

File: aries-site/src/pages/components/notifications.mdx [hunk 19]
```diff
@@ -0,0 +1,129 @@
+import { StateVsStatusDiagram } from '../../examples';
+import { Box, Text } from 'grommet';
+
+### What are they for?
+
+Notifications build trust by providing users confidence and assurance that they are informed with timely, relevant status about their systems and tasks.
+
+## Taxonomy 
+
+To standardize our language on Notifications across platforms, be sure to familiarize yourself with this taxonomy:
```
Comment (halocline) [inline aries-site/src/pages/components/notifications.mdx:10]:
```suggestion
To standardize language for Notifications across HPE projects, be sure to familiarize yourself with this taxonomy:
```

File: aries-site/src/pages/components/notifications.mdx [hunk 20]
```diff
@@ -0,0 +1,129 @@
...
+
+A state is the current stage of a process.
+
+A status is the condition of the state. Status can also be thought of as a pointer to a state’s “health”. 
+
+Here is an example of a Toast Notification that illustrates their differences:
+
+<Box>
+    <StateVsStatusDiagram />
+</Box>
```
Comment (halocline) [inline aries-site/src/pages/components/notifications.mdx:105]: Is this Box needed?
```suggestion
<StateVsStatusDiagram />
```

File: aries-site/src/pages/components/status-indicator.mdx (hunk 4 above)
Comment (halocline) [reply aries-site/src/pages/components/status-indicator.mdx:46]: Is there a relevant place to link within the Voice & Tone page for this?

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 21]
```diff
@@ -0,0 +1,100 @@
...
+- Requires user action or input
+- Batch updates
+
+Due to their limited use cases, toasts are advised to be used sparingly or even substituted entirely for a different type of notification. In no way are toasts mandatory elements for a platform’s notification system.
+
+## Anatomy 
+
+<Box pad="small">
+  <ToastDiagram />
+</Box>
```
Comment (halocline) [inline aries-site/src/pages/components/toast-notifications.mdx:45]: Can we eliminate Box, add pad to ToastDiagram and ...rest in ToastDiagram implementation?

File: aries-site/src/data/structures/components.js (hunk 10 above)
Comment (amandadupell) [reply aries-site/src/data/structures/components.js:440]: yes these could be used outside of the context of notifications so the recommended change of language makes sense to me!

File: aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js (hunk 18 above)
Comment (amandadupell) [reply aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js:79]: makes sense to me!

File: aries-site/src/examples/components/toast-notifications/ToastNotification.js (hunk 14 above)
Comment (amandadupell) [reply aries-site/src/examples/components/toast-notifications/ToastNotification.js]: moved to be in the notifications folder in `src/examples/components/notifications` and renamed the file to be `Notification.js`

File: aries-site/src/pages/components/notifications.mdx [hunk 22]
```diff
@@ -0,0 +1,55 @@
+import { StateVsStatusDiagram, TaxonomyTable } from '../../examples';
+import { Box, Text } from 'grommet';
+
+### What are they for?
```
Comment (taysea) [inline aries-site/src/pages/components/notifications.mdx:4]: Since this comes directly after the h1, it should be an h2
```suggestion
## What are they for?
```

File: aries-site/src/examples/components/notifications/TaxonomyTable.js [hunk 23]
```diff
@@ -0,0 +1,140 @@
...
+        persistence: '',
+        placement: 'Inline',
+        insight: '',
+    },
+];
+
+export const columns = [
+    {
+        property: 'type',
+        header: <Text weight="bold">Type</Text>,
```
Comment (taysea) [inline aries-site/src/examples/components/notifications/TaxonomyTable.js:100]:
```suggestion
        header: <Text color="text-strong" weight="bold">Type</Text>,
```
Can we add color="text-strong" to all these table headings to align with way we style datatable headings?

File: aries-site/src/pages/components/status-indicator.mdx [hunk 24]
```diff
@@ -0,0 +1,54 @@
...
+These are some typical situations in which each color could be used: 
+
+- A critical status used to alert a device failure
+- A warning status used to notify the user of an expiring license
+- A normal status used to communicate that an update has successfully completed
+- An unknown status used to indicate that a page cannot be found
+
+### Icons and Shapes
+
+Icons and shapes are used to represent the notification’s level of severity to users. Here are two sets of icons and shapes created for Statuses found within the [HPE Design System Icons](https://design-system.hpe.design/foundation/icons) page.
```
Comment (taysea) [inline aries-site/src/pages/components/status-indicator.mdx:33]: Can we remove the comment around "two sets of icons" since it seems we're just using one set?

File: aries-site/src/pages/components/status-indicator.mdx [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+
+<StatusBox />
+
+*** Remember that WCAG 2.1 guidlines states that at least three out of four elements must be present, therefore, this sets will not pass accessibility requirements if it is not paired with another element such as well-written and relevant content.
```
Comment (taysea) [inline aries-site/src/pages/components/status-indicator.mdx:37]:
```suggestion
*** Remember that WCAG 2.1 guidelines states that at least three out of four elements must be present, therefore, this set will not pass accessibility requirements if it is not paired with another element such as well-written and relevant content. 
```

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 26]
```diff
@@ -0,0 +1,98 @@
...
+
+Aside from this, all toasts have a fixed width size at Medium (384px) while the height changes based off the amount of content. 
+
+### Status Indicator
+
+Avoid using toasts for very critical information or alerts. Users may accidentally dismiss or entirely miss a toast, so it is best to avoid this case for important information. Additionally, avoid using toasts, critical or not, if it requires a user action. 
+
+The status indicators used for toasts are strictly for OK, Warning, and Unknown statuses.
+
+<Box direction="row" gap="medium" pad={{bottom: 'medium'}}>
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:59]: Can we do similar thing here to what you did on the other page with the labels on the icons?

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 27]
```diff
@@ -0,0 +1,98 @@
...
+    <StatusGoodSmall color="status-ok" />
+    <StatusWarningSmall color="status-warning" />
+    <StatusUnknownSmall color="status-unknown" />
+</Box>
+
+You can read more guidance about status indicators [here on this page](/components/status-indicator).
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:65]: Maybe rephrase so that link title is more informative
```suggestion
You can read more guidance about [status indicators](/components/status-indicator).
```

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 28]
```diff
@@ -0,0 +1,98 @@
...
+
+### Content (Title + Message)
+
+All toasts have content which is comprised of a title and a message. However, the content only requires a title, not necessarily a message. The title and message should be distinguishable from each other by the weight of the text. The status title and message should be distinguishable by the weight of the text.
+
+1. Amount of Information
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:71]:
```suggestion
**1. Amount of Information**
```

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 29]
```diff
@@ -0,0 +1,98 @@
...
+
+Toast notifications should not have too much information inside of them. Additionally, each notification should only revolve around one idea or subject. Because a toast notification is only present on the screen for a short amount of time, a user using assistive technology needs to have an understanding of what information is in the notification. Therefore, the content should be minimal and direct.
+
+For best practice, keep the amount of information to three lines or under. This three line limit either can be a combination of both the title and the message or a three-lined title. If the information is heavy and long, consider using a different type of notification other than toast. Never truncate text within a toast. The main point is to keep the information concise and clear.
+
+2. Interactive Content
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:77]:
```suggestion
**2. Interactive Content**
```

File: aries-site/src/pages/components/toast-notifications.mdx [hunk 30]
```diff
@@ -0,0 +1,98 @@
...
+
+Avoid placing interactive elements as this violates WCAG 2.1 guidelines, like links and buttons, inside of a toast. These elements may be difficult to access with a screen reader and other assistive technology, so it is best to avoid complex toasts.
```
Comment (taysea) [inline aries-site/src/pages/components/toast-notifications.mdx:79]:
```suggestion
To remain compliant with WCAG 2.1 guidelines, avoid placing interactive elements, like links and buttons, inside of a toast. These elements may be difficult to access with a screen reader and other assistive technology, so it is best to avoid complex toasts.
```

**Comment by britt6612** [comment]
LGTM! I also think we should all get together and decide where this should live before merging. second this 👆🏻

**Comment by britt6612** [comment]
After a discussion about the Notifications page:

- [ ] Move `status Indicators` to live under templates 
- [ ] Move `Toast Notifications` to live under templates

Make the following changes to the Notifications page:

- [ ] Change title to be singular so `Notification`
- [ ] Add Variants to Notification example and link to templates (put this before roadmap)
- [ ] Move `Notification Systems` up above variants

**Comment by britt6612** [comment]
LGTM once the merge conflict is fixed!

File: aries-site/src/examples/components/index.js [hunk 31]
```diff
@@ -36,6 +37,7 @@ export * from './tabs';
 export * from './table';
 export * from './textarea';
 export * from './textinput';
+export * from './toast-notifications';
```
Comment (taysea) [inline aries-site/src/examples/components/index.js:40]: should we move these to the templates folder since that is where "Toast Notifications" page is?

File: aries-site/src/examples/components/notifications/StateVsStatusDiagram.js [hunk 32]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { ToastPreview } from '../../../examples/cardPreviews';
+import { Box, Text, Diagram, Stack } from 'grommet';
+
+const color = 'black';
```
Comment (taysea) [inline aries-site/src/examples/components/notifications/StateVsStatusDiagram.js:5]:
```suggestion
const color = 'text-strong';
```

File: aries-site/src/examples/components/toast-notifications/ToastDiagram.js [hunk 33]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Box, Diagram, Stack } from 'grommet';
+import { ToastPreview } from '../../../examples/cardPreviews';
+
+const color = 'black';
```
Comment (taysea) [inline aries-site/src/examples/components/toast-notifications/ToastDiagram.js:5]:
```suggestion
const color = 'text-strong';
```

File: aries-site/src/pages/components/notification.mdx [hunk 34]
```diff
@@ -0,0 +1,71 @@
...
+## Variants
+
+There are a couple "types" of the Notification component depending on use case.
+
+### Toast Notification
+
+Toast notifications are used for low attention status messages or updates that do not require user action. For more thorough guidance on how and when to properly make use of a toast, access the full page on [Toast Notifications](/templates/toast-notifications).
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js"
```
Comment (taysea) [inline aries-site/src/pages/components/notification.mdx:52]: If we update the folder where the examples live, we should also make sure to update the code links

**Review by taysea — COMMENTED** [review]
Tiny internal code structure comments, but otherwise looks good!

**Review by taysea — APPROVED** [review]
LGTM! Thanks for being patient with all the feedback.

**Review by halocline — APPROVED** [review]
Great job pushing this through.
